test/integration/variable_test.rb

```diff
@@ -80,10 +80,15 @@ def test_hash_with_default_proc
     assigns['test'] = 'Tobi'
     assert_equal 'Hello Tobi', template.render!(assigns)
     assigns.delete('test')
-    e = assert_raises(RuntimeError) do
```

Comment:
Hmm. Why do we not want this to raise anymore? The test is using `render!` after all.